hazelcast/src/main/java/com/hazelcast/internal/eviction/EvictionConfiguration.java

```diff
@@ -29,11 +31,11 @@
     EvictionStrategyType getEvictionStrategyType();
 
     /**
-     * Gets the type of eviction policy.
+     * Gets the eviction policy.
      *
-     * @return the type of eviction policy
+     * @return the eviction policy
      */
-    EvictionPolicyType getEvictionPolicyType();
+    EvictionPolicy getEvictionPolicy();
```

Comment:
public api and should be kept as is? see [EvictionConfig](https://github.com/hazelcast/hazelcast/blob/master/hazelcast/src/main/java/com/hazelcast/config/EvictionConfig.java#L39)